### PR TITLE
added documentation about Eclipse IoT Marketplace integration

### PR DESCRIPTION
**Pull request by kaikreuzer**
@ThomDietrich I will need that page in place for my release annoucement tomorrow, so please review with some priority - thanks :-)

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: configuration/eclipseiotmarket.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: documentation
+title: Eclipse IoT Marketplace
+---
+
+{% include base.html %}
+
+# Eclipse IoT Marketplace Integration
+
+The <a href="https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/title">Eclipse IoT Marketplace</a> hosts third-party add-ons that can be browsed and installed to openHAB through this integration.
```
Comment (BClark09) [inline configuration/eclipseiotmarket.md]: It works as html but the rest of the document is written with markdown, including the same link further below.

File: configuration/eclipseiotmarket.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+To do so, you can add a configuration file `conf/services/marketplace.cfg` with content like:
+
+```
+# whether to show bindings from the marketplace (default is true)
+org.eclipse.smarthome.marketplace:bindings=false
+
+# whether to show rule templates from the marketplace (default is true)
+org.eclipse.smarthome.marketplace:ruletemplates=true
+
+# possible values for maturiy are
```
Comment (BClark09) [inline configuration/eclipseiotmarket.md]: maturiy >> maturity

**Comment by kaikreuzer** [comment]
Thanks @BClark09, both fixed.

File: addons/io.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ openHAB supports services that enable integration with various technologies that
       </td>
       <td>
         <p>
-        The <a href="https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/title">Eclipse IoT Marketplace</a> hosts third-party add-ons that can be browsed and installed to openHAB through this integration.
+        The [Eclipse IoT Marketplace](https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/title) hosts third-party add-ons that can be browsed and installed to openHAB through this integration.
```
Comment (BClark09) [inline addons/io.md]: Wrong one :P This document's all html.

**Comment by kaikreuzer** [comment]
outch.

**Comment by kaikreuzer** [comment]
next try pushed :-)

**Comment by BClark09** [comment]
:ok_hand:

File: configuration/eclipseiotmarket.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Installation
+
+Install the market integration from Add-ons → Misc → Eclipse IoT Market.
+
+![iot marketplae addon](images/eclipseiotmarket_addons.png)
+
+## Usage
+
+When you now refresh your browser, you will see new entries in the "Bindings" tab of the Add-ons menu. As the new entries come with logos and descriptions, the card layout (instead of the list layout, can be selected in the top right) is best for browsing the entries. To filter the view to show only third-party add-ons from the marketplace, you can simply type "market" in the search bar:
```
Comment (ThomDietrich) [inline configuration/eclipseiotmarket.md]: If somebody already installed the add-on and wants to know how to use it, the first sentence here is a bit strange. How about:

> Whit the Eclipse IoT Marketplace installed, you will see additional entries in the "Bindings" tab of the Add-ons menu.

File: configuration/eclipseiotmarket.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+![entries](images/eclipseiotmarket_listings.png)
+
+Simply use the install/uninstall buttons to add or remove add-ons from your installation.
+Please note that after a system upgrade the installed add-ons are lost and you will need to re-install them. This behavior will be changed in future.
```
Comment (ThomDietrich) [inline configuration/eclipseiotmarket.md]: This should deserve its own paragraph and a something like **Attention:** `

File: configuration/eclipseiotmarket.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+As an alternative installation method, you can browse the catalog on the [Eclipse Marketplace website](https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/title) and use the "Drag to Install" feature by dragging the "Install" button from the web page to your add-ons page of the Paper UI.
```
Comment (ThomDietrich) [inline configuration/eclipseiotmarket.md]: Own section?

File: configuration/eclipseiotmarket.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+## Configuration
+
+The marketplace integration can be configured to only provide add-ons of certain types (Note: Currently only bindings are supported!) and of a certain maturity level.
+
+To do so, you can add a configuration file `conf/services/marketplace.cfg` with content like:
+
+```
```
Comment (ThomDietrich) [inline configuration/eclipseiotmarket.md]: Ideally all markdown code blocks should have an explicit language tag. This one here would be "conf"

**Comment by kaikreuzer** [comment]
> You've organized it under configuration, maybe because the restdocs file is also located there. 

Correct.

> Would the addons folder not be more appropriate?

Directly in addons? So far, we have only the menu pages there, nothing else. And we also have rules-ng under configuration and others. Also note that this is not a "classical" add-on as it registers deep in the system, once installed. So I would tend to leave it where it is.

**Comment by ThomDietrich** [comment]
> So I would tend to leave it where it is.

okay.

**Comment by kaikreuzer** [comment]
All other comments should be addressed now.

File: configuration/eclipseiotmarket.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+# Eclipse IoT Marketplace Integration
+
+The [Eclipse IoT Marketplace](https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/title) hosts third-party add-ons that can be browsed and installed to openHAB through this integration.
+
+## Installation
+
+Install the market integration from Add-ons → Misc → Eclipse IoT Market.
+
+![iot marketplae addon](images/eclipseiotmarket_addons.png)
```
Comment (ThomDietrich) [inline configuration/eclipseiotmarket.md]: c

**Comment by ThomDietrich** [comment]
>  	bought a "c" 
:laughing: lgtm

**Pull request merged**